Source/missiles.h

```diff
@@ -57,34 +57,39 @@ struct MissilePosition {
 	}
 };
 
-/*
- *      W  sW  SW   Sw  S
- *              ^
- *     nW       |       Se
- *              |
- *     NW ------+-----> SE
- *              |
- *     Nw       |       sE
- *              |
- *      N  Ne  NE   nE  E
+/**
+ * Represent a more fine-grained direction than the 8 value Direction enum.
+ *
+ * This is mainly used in Hellfire specific rendering code to draw sprites for "half-winds" on a 16-point compass.

```

Comment:
This isn't really true afaik, the only place where it was specifically used in Hellfire is `AddFirenova()`. Instead it would be more accurate to say that it is exclusively used for rendering projectile missile directions.